config/routes.rb

```diff
@@ -16,7 +16,9 @@
   put 'user_reports/undo_delete/:id' => 'user_reports#undo_delete', as: :undo_delete_users_report
 
   resources :pages, only: [:index, :new, :create, :edit]
-  resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy]
+  resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy] do
+    get 'search', on: :collection
```

Comment:
what's the advantage of this?  we don't have it for our organisation search at the moment ...